### DISCUSSION TITLE
Daily timeline summary

### DISCUSSION THREAD

**Discussion by MrDiamond10** [Freika/dawarich#489] [Ideas]
Hello, I really like this project. I started to use it on daily basis. THANK YOU for your work.
So, there is my question from pracital life. Is there possible to show my daily timeline? Yes, I see visited Cities, but I would like to know time also, not only date. 
For example, I start drive by car from city A through city B to city C, then back to city A. So, in timeline I would like to see where I was during the day, something like summary. Something like that  9:00 - City A -> 10:30 - City B -> 13:11 - City C -> 16:23 - City A. 
Is it possible, please? Because I was not able to find summary like that and it would be really helpful for tracking of my car trips. 
Thanks
Best regards

**Comment by Freika** [comment]
I want to work on summaries in the future, to cover daily, monthly and yearly timespans, so yes, it will be a thing :)

**Reply by Hulabaloon** [reply]
This would be amazing and would mean I can ditch Google Maps timeline for good. Being able to go to a specfic day and follow through a timeline of what I did and where I went would be brilliant. Thanks for all your work on this.
